authentication/serializers.py

```diff
@@ -2,13 +2,18 @@
 
 from django.contrib.auth import authenticate, get_user_model
 from django.core.exceptions import ValidationError
+from django.conf import settings as custom_settings
 from djoser.conf import settings
```

Comment:
why not `from django.conf import settings as django_settings` and `from djoser.conf import settings as auth_settings` or `from djoser.conf import settings as djoser_settings`

This is much clearer in code when the imports says what about they are.

Reply:
Thank you for feedback. I've changed names for settings to django_settings and djoser_settings